### ISSUE TITLE
Examples box doesn't actually have any examples

### ISSUE THREAD

**Issue by Lyxal** [ZippyMagician/ayr#3]
I click on the examples box, and nothing drops down.

**Comment by ZippyMagician** [comment]
Yeah I just haven’t added anything yet, I grabbed this code from my Arn online interpreter and haven’t fixed everything yet :p.
